Product/Production/Common/Properties/src/main/resources/assertioninfo.properties

```diff
@@ -19,3 +19,4 @@ AccessPolicyConsent=urn:oid:1.2.3.4
 InstanceAccessPolicyConsent=urn:oid:1.2.3.4.123456789
 SamlAuthInstant=2009-04-16T13:15:39Z
 AuthContextClassRef=urn:oasis:names:tc:SAML:2.0:ac:classes:X509
+SamlIssuerName=CN=SAML User%OU=SU%O=SAML User%L=Los Angeles%ST=CA%C=US

```

Comment:
Define the property as 
SamlIssuerName=CN=SAML User\,OU=SU\,O=SAML User\,L=Los Angeles\,ST=CA\,C=US 
and you don't need to do anything else.